### PR TITLE
3088 hsapi

### PR DESCRIPTION
**Pull request by sblack-usu**
### Pull Request Checklist: 
- [x] Positive Test Case Written by Dev

- [x] Automated Testing

- [x] Sufficient User and Developer Documentation

- [ ] Passing Jenkins Build

- [ ] Peer Code review and approval

### Positive Test Case
1. Update "rights" via the rest api
2. Add a file via the rest api and verify the response body includes file_id

### CODE REVIEW

**Comment by hydrobuild** [comment]
Can one of the admins verify this patch?

**Comment by cuahsimarko** [comment]
Not sure it tests failed as a part of the CUAHSI network outage yesterday, but will need to rerun and fix if necessary.

File: hs_core/models.py [hunk 1]
```diff
@@ -3525,7 +3525,7 @@ def parse_for_bulk_update(cls, metadata, parsed_metadata):
             parsed_metadata.append({"language": {"code": metadata.pop('language')}})
 
         if 'rights' in keys_to_update:
-            parsed_metadata.append({"rights": {"statement": metadata.pop('rights')}})
```
Comment (hyi) [inline hs_core/models.py]: @sblack-usu Removing this ```statement``` nesting might potentially break landing page rights display. For example, https://github.com/hydroshare/hydroshare/blob/344f4590c8615fd7ffd67e4c04930e23b6504147/theme/templates/resource-landing-page/citation.html#L34

**Comment by sblack-usu** [comment]
retest this, please

File: hs_core/models.py (hunk 1 above)
Comment (sblack-usu) [reply hs_core/models.py]: this is only for the rest api and was incorrectly implemented.  The internal urls pass in statement and url.

**Comment by sblack-usu** [comment]
retest this, please

**Comment by phuongdm** [comment]
retest this, please

**Comment by phuongdm** [comment]
retest this, please

**Comment by phuongdm** [comment]
retest this, please

**Comment by phuongdm** [comment]
retest this, please

**Pull request closed without merging**